tests/test_shadow_run_id.py

```
@@ -0,0 +1,304 @@
+        from core_brain.order_registry import OrderRegistry
+
+        monkeypatch.setattr(order_registry, "_CURRENT_RUN_ID", self.LIVE)
+        return OrderRegistry(db_path=tmp_path / "scoped.db", run_id=self.MINE)
+
+    def _one(self, reg, table):
+        import sqlite3
+
+        conn = sqlite3.connect(reg.db_path)
+        try:
+            conn.row_factory = sqlite3.Row
+            return [r["run_id"] for r in conn.execute(f"SELECT run_id FROM {table}")]
```

Ruff (0.16.2)

[error] Possible SQL injection vector through string-based query construction

(S608)